src/officehours_ui/templates/base.html

```diff
@@ -86,6 +86,17 @@
   <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
   <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
+  <script>
+    const logoutButton = document.querySelector('#logout-button');
+    logoutButton.addEventListener('click', () => {
+      window.localStorage.setItem("loggedIn", "false");
+    });
```

Comment:
I think this is throwing a console error when the user is logged out, since this button doesn't exist. One solution would be to wrap it in an `if` clause, like so:
```
    const logoutButton = document.querySelector('#logout-button');
    if (logoutButton !== null) {
      logoutButton.addEventListener('click', () => {
        window.localStorage.setItem("loggedIn", "false");
      });
    }
```